packages/botonic-plugin-contentful/src/nlp/stopwords/stopwords-nl.ts

```diff
@@ -0,0 +1,146 @@
+//from https://github.com/axa-group/nlp.js/blob/master/packages/lang-nl/src/stopwords-nl.js
+export const nlDefaultStopWords = [
+  'aan',
+  'af',
+  'al',
+  'alles',
+  'als',
+  'altijd',
+  'andere',
+  'ben',
+  'bij',
+  'daar',
+  'dan',
+  'dat',
+  'de',
+  'der',
+  'deze',
+  'die',
+  'dit',
+  'doch',
+  'doen',
+  'door',
+  'dus',
+  'een',
+  'eens',
+  'en',
+  'er',
+  'ge',
+  'geen',
+  'geweest',
+  'haar',
+  'had',
+  'heb',
+  'hebben',
+  'heeft',
+  'hem',
+  'het',
+  'hier',
+  'hij',
+  'hoe',
+  'hun',
+  'iemand',
+  'iets',
+  'ik',
+  'in',
+  'is',
+  'ja',
+  'je ',
+  'kan',
+  'kon',
+  'kunnen',
+  'maar',
+  'me',
+  'meer',
+  'men',
+  'met',
+  'mij',
+  'mijn',
+  'moet',
+  'na',
+  'naar',
+  'niet',
+  'niets',
+  'nog',
+  'nu',
+  'of',
+  'om',
+  'omdat',
+  'ons',
+  'ook',
+  'op',
+  'over',
+  'reeds',
+  'te',
+  'tegen',
+  'toch',
+  'toen',
+  'tot',
+  'u',
+  'uit',
+  'uw',
+  'van',
+  'veel',
+  'voor',
+  'want',
+  'waren',
+  'was',
+  'wat',
+  'we',
+  'wel',
+  'werd',
+  'wezen',
+  'wie',
+  'wij',
+  'wil',
+  'worden',
+  'zal',
+  'ze',
+  'zei',
+  'zelf',
+  'zich',
+  'zij',
+  'zijn',
+  'zo',
+  'zonder',
+  'zou',
+  'a',
```

Comment:
we don't have single letters & numbers in other languages

Reply:
okay, I will delete them :smile: